src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.SearchResult.cs

```diff
@@ -15,21 +15,22 @@ internal abstract partial class AbstractNavigateToSearchService
     {
         private class SearchResult : INavigateToSearchResult
         {
-            public string AdditionalInformation => _lazyAdditionalInfo.Value;
-            public string Name => _declaredSymbolInfo.Name;
             public string Summary { get; }
-
             public string Kind { get; }
             public NavigateToMatchKind MatchKind { get; }
             public INavigableItem NavigableItem { get; }
-            public string SecondarySort { get; }
             public bool IsCaseSensitive { get; }
             public ImmutableArray<TextSpan> NameMatchSpans { get; }
 
+            public string Name => _declaredSymbolInfo.Name;
+            public string AdditionalInformation => _lazyAdditionalInfo.Value;
+
             private readonly Document _document;
             private readonly DeclaredSymbolInfo _declaredSymbolInfo;
             private readonly Lazy<string> _lazyAdditionalInfo;
 
+            private string _secondarySort;
```

Comment:
// Lazily compute secondary-sort string as VS doesn't need it unless it is tiebreaking between two items that are otherwise identical.